sys-devel/ucpp/ucpp-1.3.4.ebuild

```diff
@@ -1,12 +1,12 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
 
 inherit eutils
 
 DESCRIPTION="A quick and light preprocessor, but anyway fully compliant to C99"
-HOMEPAGE="https://code.google.com/p/ucpp/"
+HOMEPAGE="https://gitlab.com/scarabeusiv/ucpp"
 SRC_URI="http://dev.gentooexperimental.org/~scarabeus/${P}.tar.xz"
```

Comment:
Does upstream now release on Gitlab (https://gitlab.com/scarabeusiv/ucpp/releases)? If so, `SRC_URI` should be updated to match.

Reply:
While there is release for 1.3.4, the gitlab version is autotools based (contents of src are the same), and requires rewriting the ebuild. It would make more sense to make a bump for 1.3.5 (and removing the 9999).
This goes beyond the scope of the bug. But if the maintainers are interested, I could make an attempt at this.

Reply:
Ah, I see what you mean, (1.3.4 on Gitlab)[https://gitlab.com/scarabeusiv/ucpp/-/archive/1.3.4/ucpp-1.3.4.tar.gz] requires `autoreconf` to generate the configure script.

Fair enough, in that cause it may be best to make a bump for 1.3.5 instead and keep the scope of this patch small as you suggest.